x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -426,26 +434,28 @@ const EventDetailsComponent: React.FC<Props> = ({
     }),
     [rawEventData]
   );
-
+  const responseActionsTab = useResponseActionsTab({
+    rawEventData: rawEventData as RawEventData,
+    ...(detailsEcsData !== null ? { ecsData: detailsEcsData } : {}),
+  });
   const osqueryTab = useOsqueryTab({
     rawEventData: rawEventData as RawEventData,
     ...(detailsEcsData !== null ? { ecsData: detailsEcsData } : {}),
   });
-
   const endpointResponseActionsTab = useEndpointResponseActionsTab({
     rawEventData: rawEventData as RawEventData,
   });
 
+  const responseActionsTabs = useMemo(() => {
+    return commonResponseActionsTabEnabled
+      ? [responseActionsTab]
+      : [osqueryTab, endpointResponseActionsTab];
```

Comment:
potentially if we decide to use the common tab - we could get rid of these 2 tabs.